tests/sles4sap/sap_deployment_automation_framework/deploy_sap_systems.pm

```diff
@@ -18,6 +18,41 @@ use sles4sap::console_redirection;
 use serial_terminal qw(select_serial_terminal);
 use testapi;
 
+=head1 SYNOPSIS
+
+Executes SDAF deployment of SAP systems infrastructure (SUT VM and related resources).
+No APP installation or configuration is done at this point.
+
+TFVARS file is composed using OpenQA paramerer B<`SDAF_DEPLOYMENT_SCENARIO`>.
```

Comment:
s/paramerer/setting/

Reply:
In case it's not clear, I'm only commenting on this due to the typo in `paramerer`, but I tend to prefer to refer to openQA variables as `settings` as that's how they're referred to as in the test results. I'm fine with `setting`, `variable` or `parameter`, but not with the typo. :P

Reply:
Oh thanks. 
I prefer test setting as well even though I have used other term. 
Fixed.